src/FirePokemon.java

```diff
@@ -0,0 +1,39 @@
+package src;
+
+public abstract class FirePokemon extends Pokemon {
+
+    private String type;
+    public FirePokemon(double healthPoint, double experiencePoints) {
```

Comment:
Let er op dat je met deze constructor null waardes krijgt voor je type, attack, defence, etc.. Dat is meestal niet iets wat je wilt!